docker/Dockerfile

```diff
@@ -71,10 +59,11 @@ COPY --from=stage_build /emsdk /emsdk
 # Fallback in case Emscripten isn't activated.
 # This will let use tools offered by this image inside other Docker images
 # (sub-stages) or with custom / no entrypoint
-ENV EMSDK_NODE=/emsdk/node/12.18.1_64bit/bin/node
-ENV EMSDK=/emsdk
-ENV EM_CONFIG=/emsdk/.emscripten
-ENV PATH="${EMSDK}:${EMSDK}/upstream/emscripten:${EMSDK}/upstream/bin:emsdk/node/12.18.1_64bit/bin:${PATH}"
+ENV EMSDK=/emsdk \
+    EM_CACHE=/emsdk/upstream/emscripten/cache \
+    EM_CONFIG=/emsdk/.emscripten \
+    EMSDK_NODE=/emsdk/node/12.18.1_64bit/bin/node \
+    PATH="/emsdk:/emsdk/upstream/emscripten:/emsdk/upstream/bin:/emsdk/node/12.18.1_64bit/bin:${PATH}"
```

Comment:
I assume this is better from a docker perspective because it generated fewer intermediate images?    Otherwise I think its less readable that the existing on-line-per-variable form.

Reply:
That's right, I did this to minimize the intermediate layers. I can undo this if you'd prefer it to be more readable.

As an aside,
I also added the missing `EM_CACHE` environment variable here. I hope it's OK.

Reply:
I don't think its necessary.   emscripten these days defaults to using an embedded/local cache.   The EM_CACHE settings in emsdk for legacy/old versions that don't do this.

Reply:
Ah, I see. I thought it was still in use since `emsdk_env.sh` script sets this environment variable.
```bash
STEP 8: RUN echo "## Aggressive optimization: Remove debug symbols"     && cd ${EMSDK} && . ./emsdk_env.sh     && strip -s `which node`     && rm -fr ${EMSDK}/upstream/emscripten/tests     && rm -fr ${EMSDK}/upstream/fastcomp     && find ${EMSDK}/upstream/bin -type f -exec strip -s {} + || true     && echo "## Done"
## Aggressive optimization: Remove debug symbols
Adding directories to PATH:
PATH += /emsdk
PATH += /emsdk/upstream/emscripten
PATH += /emsdk/node/12.18.1_64bit/bin

Setting environment variables:
EMSDK = /emsdk
EM_CONFIG = /emsdk/.emscripten
EM_CACHE = /emsdk/upstream/emscripten/cache
EMSDK_NODE = /emsdk/node/12.18.1_64bit/bin/node
```

Nevertheless, I will remove it.

Reply:
Yeah.. emsdk has to be conservative because it can install old versions too.. where the cache is not in the new default location.